src/test/resources/fixtures/hmh/hmh-interacted/development/caliperEventAssessmentItemCompleted.json

```diff
@@ -0,0 +1,186 @@
+                "dateToShow": "2019-06-10T15:53:10.418Z",
+                "dateToStartOn": "2019-06-10T15:53:10.418Z",
+                "dateToSubmit": "2019-06-10T15:53:10.418Z",
+                "maxAttempts": 0,
+                "maxSubmits": 0,
+                "maxScore": 0
+            },
+            "eventTime": "2019-06-10T14:53:11.812Z",
+            "generated": {},
+            "target": {
+                "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+                "id": "http://rce.localhost/content/book/OPS/s9ml/cards/el_uat_content_samp_011.xhtml",
+                "type": "Frame",
+                "name": "Flip Cards",
+                "description": null,
+                "dateCreated": null,
+                "dateModified": null,
+                "extensions": {
+                    "programId": null,
+                    "programName": null,
+                    "contentRootId": "080c58111f020b9418f824970ffd7bb5392",
+                    "contentRootType": "pubId",
+                    "contentVersionId": 0,
+                    "userRole": "student",
+                    "contentItemId": "d5763388b980473ebbfce7876b23e345"
+                },
+                "mediaType": null,
+                "creators": [],
+                "keywords": [],
+                "learningObjectives": [],
+                "isPartOf": {
+                    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+                    "id": "http://rce.localhost/content/book/OPS/s9ml/cards/el_uat_content_samp_001.xhtml",
+                    "type": "DigitalResource",
+                    "name": "Module 1: QA Review",
+                    "description": null,
+                    "dateCreated": null,
+                    "dateModified": null,
+                    "extensions": {},
+                    "mediaType": null,
+                    "creators": [],
+                    "keywords": [],
+                    "learningObjectives": [],
+                    "isPartOf": {
+                        "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+                        "id": "http://rce.localhost/digitalresource/080c58111f020b9418f824970ffd7bb5392",
+                        "type": "DigitalResource",
+                        "name": "Experience Library Interaction QA Review",
+                        "description": null,
+                        "dateCreated": null,
+                        "dateModified": null,
+                        "extensions": {},
+                        "mediaType": null,
+                        "creators": [],
+                        "keywords": [],
+                        "learningObjectives": [],
+                        "isPartOf": {
+                            "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+                            "id": "NoProgrammeId",
+                            "type": "DigitalResource",
+                            "name": "NoProgramme",
```

Comment:
simliar to comments above - 'noProgramme' etc... to me it looks like isPartOf should be deleted entirely because there is nothing in here?

Reply:
the same above, this is part of the RCE event object